frontend/package.json

```diff
@@ -9,14 +9,13 @@
     "all-checks": "npm run lint && npm run ts:check && npm run test && npm run build -- --no-lint",
     "build": "next build",
     "dev": "NEW_RELIC_ENABLED=false next dev",
-    "dev:nr": "NODE_OPTIONS='-r @newrelic/next' next dev",
```

Comment:
turns out this doesn't work for some reason, and we likely wouldn't ever need to use it anyway